include/ArduPilotPlugin.hh

```diff
@@ -97,7 +96,7 @@ class GZ_SIM_VISIBLE ArduPilotPlugin:
   public: ArduPilotPlugin();
 
   /// \brief Destructor.
-  public: ~ArduPilotPlugin();
+  public: ~ArduPilotPlugin() final = default;
```

Comment:
Probably not a problem here, but there can be some nasty side effects in shared libraries when defining virtual methods inline in headers. What happens is that each DLL using the header creates a copy of the destructor, so it is undefined which particular one gets called. Worse, a DLL unloading can destroy objects that are still supposed to be in scope. Biggest issue is with templates and base classes, but I now avoid this after being burned. Was a problem in the core Gazebo libraries for some time - now corrected.